model/indexes.go

```diff
@@ -0,0 +1,52 @@
+package model
+
+// +build ignore
+
+import (
+	"github.com/mongodb/anser/bsonutil"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// Indexes for "perf_results" collection.
+const (
+	PerfResultsCreatedCompletedIndex = bson.D{{perfCreatedAtKey, 1}, {perfCompletedAtKey, 1}}
```

Comment:
I was thinking something more like: 

type SystemIndexes struct {
      Spec bson.D
      Options bson.D
      Collection string
}

and then a function that just returns a slice of those objects with the data that you have here.